manifest/product-config.json

```
@@ -450,14 +450,24 @@
             "start_build": 1
         },
         "manifest/3.1.xml": {
-            "release": "3.1.3.1",
-            "release_name": "Couchbase Sync Gateway 3.1.3.1",
+            "release": "3.1.4",
+            "release_name": "Couchbase Sync Gateway 3.1.4",
             "production": true,
             "interval": 120,
             "go_version": "1.21.4",
             "trigger_blackduck": true,
             "start_build": 1
         },
+        "manifest/3.1/3.1.3.1.xml": {
+            "do-build": false,
+            "release": "3.1.3.1",
+            "release_name": "Couchbase Sync Gateway 3.1.3.1",
+            "production": true,
+            "interval": 1440,
+            "go_version": "1.21.4",
+            "trigger_blackduck": true,
+            "start_build": 7
```

3.1.3.1 has 2 builds (3.1.3 has 6)

```suggestion
            "start_build": 3
```